dxr/templates/file.html

```diff
@@ -37,7 +37,11 @@
           <ul>
             {% for item in section['items'] %}
               <li>
-                <a href="{{ item.href }}" title="{{ item.title }}" class="{{ item.icon }} icon">{{ item.title }}</a>
+                {% if 'template' in item %}
```

Comment:
I have a slight preference for DRYing this and putting the `if` only around the template attr.